lib/src/components/NxCollapsibleItems/__tests__/NxCollapsibleItems.test.tsx

```diff
@@ -24,10 +24,6 @@ describe('NxCollapsibleItems', function() {
 
   it('is aliased as NxTreeView', function() {
     expect(NxCollapsibleItems).toBe(NxTreeView);
-  });
-
-  it('has NxCollapsibleItemsChild that is aliased as NxTreeViewChild', function() {
-    expect(NxCollapsibleItemsChild).toBe(NxTreeViewChild);
```

Comment:
This isn't quite what I meant.  We want `expect(NxCollapsibleItems.Child).toBe(NxTreeViewChild);`

Reply:
> This isn't quite what I meant. We want `expect(NxCollapsibleItems.Child).toBe(NxTreeViewChild);`

Thanks, updated it here: 0df577d035319b98ebd84048cd7bf2d151392346